Paper.tex

```diff
@@ -281,15 +281,15 @@ \subsection{The Transaction} \label{ch:transaction}
 
 \begin{equation}
 L_T(T) \equiv \begin{cases}
-(T_n, T_p, T_g, T_t, T_v, T_\mathbf{i}, T_w, T_r, T_s) & \text{if} \; T_t = \varnothing\\
-(T_n, T_p, T_g, T_t, T_v, T_\mathbf{d}, T_w, T_r, T_s) & \text{otherwise}
+(T_n, T_mathrm{p}, T_g, T_t, T_v, T_\mathbf{i}, T_w, T_r, T_s) & \text{if} \; T_t = \varnothing\\
+(T_n, T_mathrm{p}, T_g, T_t, T_v, T_\mathbf{d}, T_w, T_r, T_s) & \text{otherwise}
```

Comment:
Other elements of the tuple should be treated the same way.

Reply:
Hmm, I wonder why that wasn't picked up.

Reply:
Ah I know what I did wrong.